sources/packages/web/src/views/aest/institution/Programs.vue

```diff
@@ -40,8 +31,11 @@
           @sort="pageSortEvent($event)"
           :loading="loading"
         >
+          <template #empty>
+            <p class="text-center font-weight-bold">No records found.</p>
+          </template>
           <Column
-            field="submittedDate"
+            :field="AESTProgramFields.SubmittedDate"
```

Comment:
These program fields are going to be reused all the times that we are doing a sort on the programs, right? Why does it includes the prefix "AEST"?